core/background/controller.js

```diff
@@ -128,7 +126,7 @@ define([
 
 			if (currentSong.flags.isProcessed) {
 				playbackTimer.update(currentSong.getSecondsToScrobble());
-				replayDetectionTimer.update(duration - (Date.now() - lastSongStart) / 1000);
+				replayDetectionTimer.update(duration - (Date.now() - currentSong.metadata.startTimestamp) / 1000);
```

Comment:
Should be `Date.now() / 1000 - currentSong.metadata.startTimestamp`, since `Song.metadata.startTimestamp` is `Date.now() / 1000` on song object init.